Source/WTF/wtf/text/StringCommon.h

```diff
@@ -439,6 +440,18 @@ ALWAYS_INLINE static size_t findInner(const SearchCharacterType* searchCharacter
 template<typename CharacterType, std::enable_if_t<std::is_integral_v<CharacterType>>* = nullptr>
 inline size_t find(const CharacterType* characters, unsigned length, CharacterType matchCharacter, unsigned index = 0)
 {
+    if constexpr (sizeof(CharacterType) == 1) {
+        // For enough long string, we use memchr since it is faster empirically.
+        constexpr unsigned memchrThresholdLength = 16;
+        if (length > index && length - index >= memchrThresholdLength) {
+            const auto* result = reinterpret_cast<const CharacterType*>(memchr(characters + index, matchCharacter, length - index));
```

Comment:
`auto*` or `auto` would be fine here, no reason to repeat the `const` from the `reinterpret_cast`.

Reply:
OK, dropped `const`.